packages/api-reference/src/features/Operation/layouts/ModernLayout.vue

```diff
@@ -69,6 +69,10 @@ const labelId = useId()
       <div class="flex flex-row justify-between gap-1">
         <!-- Left -->
         <div class="flex gap-1">
+          <!-- Operation ID -->
+          <Badge v-if="config.showOperationId && operation.operationId">
```

Comment:
Should this be `font-code` as well? cc @cameronrohani

Reply:
none of the badges has font-code mono, yet, so I just made them look like the others ("deprecated", "experimental"…)